docs/guides/measure-qubits.mdx

```diff
@@ -9,16 +9,16 @@ To get information about a qubit's state, you can _measure_ it onto a [classical
 
 $$
 |q\rangle \xrightarrow{measure}\begin{cases}
-      |0\rangle (\text{outcome}+1), \text{with probability } p_0=|\langle q|0\rangle|^{2}\text{,} \\
-      |1\rangle (\text{outcome}-1), \text{with probability } p_1=|\langle q|1\rangle|^{2}\text{.}
+      0 (\text{outcome}+1), \text{with probability } p_0=|\langle q|0\rangle|^{2}\text{,} \\
+      1 (\text{outcome}-1), \text{with probability } p_1=|\langle q|1\rangle|^{2}\text{.}
     \end{cases}
 $$
 
 ## Apply a measurement to a circuit
 
 There are several ways to apply measurement to a circuit:
 
-### `QuantumCircuit.measure` method
+### `measure` method
```

Comment:
Why this change? It's useful to include the class name

Reply:
For consistency (see below) -- but it would of course also be an option to include the class in the other titles 🙂

Reply:
Got it. @beckykd I recommend putting the prefix in front of every method